src/pages/Home/ItemCard.jsx

```diff
@@ -1,17 +1,34 @@
-import React from 'react'
+import React,{useState,useEffect} from 'react';
+import { useQuery } from "@tanstack/react-query";
+import { getDashboard } from "../../api/dashboard.js";
 
 const ItemCard = (props) => {
-    
+
+  const { data:CardData, isloading } = useQuery({
```

Comment:
Don't fetch data in card. Instead you can fetch this data in `index.js` and pass the data as props

and keep queryKey as `dashboardData`, it might be more suitable